README.md

```diff
@@ -11,66 +11,122 @@ pages with [mkdocs-material](https://squidfunk.github.io/mkdocs-material/).
 
-### 3rd: Merge the PR to `master` to Deploy Changes
+### 2nd: Make a Pull Request from your branch to `develop`
+Once you have completed all of your changes, you need to have your changes reviewed. This is done by "creating a PR." PR stands for pull request.
 
-To get your changes deployed, merge it to the branch called `master`. This will
-cause automated scripts to deploy the changes to info.arxiv.org.  If you are
-unsure how to do this, ask a dev or admin to do it for you.
+- To make a new PR click the "Pull Requests" link in the header of the github page.
+
+- On the right hand side, click the blue button that says "New pull request"
+
+<kbd>![Screenshot how to compare changes in the pull request in github](/non-info/11-help-newPR-compare.png)</kbd>
+
+- Compare your changes by clicking on the "compare:develop" dropdown and type in your branch name and then select your branch.
+- This will compare your branch to the develop branch.
+- Click "Create pull request"
+
+<kbd>![Screenshot how to compare changes in the pull request in github](/non-info/12-help-newPR-compare.png)</kbd>
+
+- To the right, click on the gear icon to the right of "Reviewers".
+- A dropdown will appear and request people to review your PR.
+- If you are unsure who to ask to review your changes, check with your manager.
+
+<kbd>![Screenshot of how to select reviewers of your PR and create the PR in Github](/non-info/07-help-openPR.png)</kbd>
+
+- After selecting reviewers, click the blue "Create pull request" button.
+
+<kbd>![Screenshot of the location of the create pull request button in Github](/non-info/08-help-createPR.png)</kbd>
+
+- Reviewers will receive an email from Github prompting them to review your PR.
+  
+
+### 3rd: Merge the PR to `develop`
+- Once your PR has been reviewed and approved you can merge your PR to 'develop'.
+- Open your PR and scroll down and click on "Merge pull request".
+
+<kbd>![Screenshot of how to merge your pull request in Github](/non-info/09-help-mergePR.png)</kbd>
+
+- You will be prompted to confirm you decision, click "Confirm merge".
+
+  <kbd>![Screenshot of how to confirm merging your PR in Github](/non-info/10-help-confirmMerge.png)</kbd>
+
+- **Please note:** The above step will only put your changes on `develop` and they will not be viewable on info.arxiv.org. At this point you have only saved the edit in github.
+
+
+### 4th: Create a PR to deploy your changes to `master`
+
+- Once you have merged your PR into `develop`, click on the "Pull Requests" link in the header of the github page.
+- Next click on the "New pull request" button.
+
+<kbd>![Screenshot of how create a new PR to deploy your changes in Github](/non-info/11-help-newPR-compare.png)</kbd>
+
+- Next you will compare the changes between the `master` branch and the `develop` branch so you can create a PR to merge your changes into production.
+- Click on "base:develop" and a dropdown will open, type in "master" and select "master".
```

Comment:
As I mentioned above, I don't think the develop->master merge should be part of the discussion here. If we do this, we could merge directly to master, and if multiple people create dev->master PRs that can also become a mess.